src/app/containers/Disclaimer/index.test.jsx

```diff
@@ -6,11 +6,26 @@ import { ToggleContextProvider } from '#contexts/ToggleContext';
 
 import DisclaimerComponent from '.';
 
+const DISCLAIMER_FIXTURE = {
+  para1: 'Приложение Русской службы BBC News доступно для ',
+  para2: {
+    text: 'IOS ',
+    url: 'https://apps.apple.com/us/app/%D0%BD%D0%BE%D0%B2%D0%BE%D1%81%D1%82%D0%B8-%D0%B1%D0%B8-%D0%B1%D0%B8-%D1%81%D0%B8/id504278066',
+  },
+  para3: 'и ',
```

Comment:
Does this make the space after `IOS` also part of the link, and underlined?

Should maybe be this - find to merge without it though if urgent!

```suggestion
    text: 'IOS',
    url: 'https://apps.apple.com/us/app/%D0%BD%D0%BE%D0%B2%D0%BE%D1%81%D1%82%D0%B8-%D0%B1%D0%B8-%D0%B1%D0%B8-%D1%81%D0%B8/id504278066',
  },
  para3: ' и ',
```